src/field_theory/algebraic_closure.lean

```diff
@@ -136,6 +132,8 @@ theorem is_alg_closure_iff (K : Type v) [field K] [algebra k K] :
   is_alg_closure k K ↔ is_alg_closed K ∧ algebra.is_algebraic k K :=
 ⟨λ h, ⟨h.1, h.2⟩, λ h, ⟨h.1, h.2⟩⟩
 
+attribute [instance] is_alg_closure.alg_closed
```

Comment:
Isn't this a dangerous instance? Because `is_alg_closed K` doesn't specify the `k` for which lean should search for `is_alg_closure k K`.
Should this be a local attribute?